recipes/instagram/webview.js

```diff
@@ -14,16 +14,35 @@ module.exports = Ferdium => {
 
   // https://github.com/ferdium/ferdium-recipes/blob/9d715597a600710c20f75412d3dcd8cdb7b3c39e/docs/frontend_api.md#usage-4
   // Helper that activates DarkReader and injects your darkmode.css at the same time
-  Ferdium.handleDarkMode((isEnabled, helpers) => {
+  Ferdium.handleDarkMode((isEnabled) => {
+
+    var url = new URL(window.location.href);
+    var search_params = url.searchParams;
```

Comment:
nitpick: all other variables are camelCased, whereas this alone is underscore - can you please change?

Reply:
Solved on latest commit. Thank you!